data/initrd/initrd.file_list

```diff
@@ -569,6 +570,10 @@ x etc/syslog.conf etc
 
 # additional scripts for linuxrc
 x scripts /
+# hmc exists only on s390
```

Comment:
Not entirely true (Power systems have them), but might be true enough for our purposes. Power system users don't have the same problem getting access to a DVD.